sound/hda/intel-dsp-config.c

```diff
@@ -666,7 +666,7 @@ int snd_intel_dsp_driver_probe(struct pci_dev *pci)
 		return SND_INTEL_DSP_DRIVER_LEGACY;
 	}
 
-	dev_info(&pci->dev, "DSP detected with PCI class/subclass/prog-if info 0x%06x\n", pci->class);
+	dev_info_once(&pci->dev, "DSP detected with PCI class/subclass/prog-if info 0x%06x\n", pci->class);
```

Comment:
is the _once() variant the right fix? this adds a static variable and logs it only once... so if you have a system with multiple cards (e.g. igfx + dgfx) it will be reported only for the first one.

Reply:
what would be your alternate proposal @lucasdemarchi ?

Reply:
I'm not familiar with sof codebase to have a proposal, which would probably involve some refactoring of the code to log it in a different place... just wanted to point out that _once() is global, and not per device. Maybe just leave it as is too.

Reply:
I demoted this to dev_dbg(), the log is not very useful in the first place any longer.

All other logs are for single-DSP configurations so dev_info_once() is just fine.